src/openrct2/ride/Ride.h

```diff
@@ -699,12 +699,12 @@ enum
 
 enum
 {
-    RIDE_DEPART_WAIT_FOR_LOAD_MASK = 7,
-    RIDE_DEPART_WAIT_FOR_LOAD = 1 << 3,
-    RIDE_DEPART_LEAVE_WHEN_ANOTHER_ARRIVES = 1 << 4,
-    RIDE_DEPART_SYNCHRONISE_WITH_ADJACENT_STATIONS = 1 << 5,
-    RIDE_DEPART_WAIT_FOR_MINIMUM_LENGTH = 1 << 6,
-    RIDE_DEPART_WAIT_FOR_MAXIMUM_LENGTH = 1 << 7
+    RIDE_DEPART_WAIT_FOR_LOAD_MASK = 0b00000111,
+    RIDE_DEPART_WAIT_FOR_LOAD = 0b00001000,
```

Comment:
Again, I appreciate the change for the mask, but not for the flags.

If the apparent inconsistency bothers you, perhaps the masks should just not be part of the enum.